Tests/PerformanceTest/VisitorPerformanceTests.swift

```diff
@@ -27,50 +27,48 @@ public class VisitorPerformanceTests: XCTestCase {
     try XCTSkipIf(ProcessInfo.processInfo.environment["SKIP_LONG_TESTS"] == "1")
     class EmptyVisitor: SyntaxVisitor {}
 
-    XCTAssertNoThrow(
-      try {
```

Comment:
If you are modifying the test cases: There’s no need for `XCTAssertNoThrow` anymore. This was only necessary at a time when test methods weren’t allowed to throw. You should be able to just remove the calls to `XCTAssertNoThrow`.